projects/plugins/jetpack/modules/sitemaps/sitemaps.php

```diff
@@ -276,15 +269,15 @@ public function callback_action_catch_sitemap_urls() {
 			}
 
 			// Catch sitemap xsl.
-			if ( preg_match( $regex['sitemap-style'], $request['sitemap_name'] ) ) {
```

Comment:
I would imagine it would be more performant to have all of the simple comparisons before any of the `preg_match`es. Any reason not to have all of them occur before the first `preg_match`?

Reply:
Oh I didn't think of that, thanks for the suggestion!

Updated in 60df508.